src/System.Diagnostics.TraceSource/tests/TraceClassTests.cs

```diff
@@ -341,9 +351,42 @@ public void TraceTest01()
         }
 
         [Fact]
```

Comment:
TargetFrameworkName follows a well formed structure that can be parsed by things like https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/src/System/Runtime/Versioning/FrameworkName.cs#L10 and can be specified by https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Runtime/Versioning/TargetFrameworkAttribute.cs. 

Where as FrameworkDescription doesn't follow any common schema or structure and changes from framework to framework and might be changed at any time. It is only useful for logs and diagnostics purposes.